administrator/language/en-GB/en-GB.com_media.ini

```diff
@@ -67,3 +68,6 @@ COM_MEDIA_TOGGLE_INFO="Toggle item info"
 COM_MEDIA_TOGGLE_LIST_VIEW="Toggle between grid and table views"
 COM_MEDIA_TOGGLE_SELECT_ITEM="Toggle select item"
 COM_MEDIA_UPDLOAD_SUCCESS="Item uploaded."
+COM_MEDIA_SHARE="Share the file"
```

Comment:
We have the policy in Joomla to alpha order the strings.

Reply:
Will fix asap